torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3235,8 +3287,34 @@ def reference_lgamma(x):
     ]
     op_db = op_db + op_db_scipy_reference
 
+for op in (('add', 'add'), ('mul', 'multiply'), ('div', 'divide')):
```

Comment:
This is clever but let's just add these ops by hand to the op_db list instead of programmatically enumerating them. I imagine that `div` will require special handling, for example.

Reply:
How do you want to handle div in this PR?

One option would be to have a domain, but allow specifying a "left domain" or "input domain" and a "right domain" or "right domain." Then divide()'s divisor could be specified to always be greater than 0 + eps, for example. That wouldn't allow generating negative divisors, however. Maybe a follow-up PR could resolve that issue, possibly by updating make_tensor() with a refinement like "bounded away from zero," or even support for more arbitrary domains that include gaps?